### PR TITLE
Make flit install succeed

### PR DESCRIPTION
**Pull request by tm-kn**
- prompts.py import Django and the Flit installation script has no Django installed.
- `Framework :: Django :: 5.1` is supposedly an invalid classifier.

### CODE REVIEW

**Comment by tomusher** [comment]
I see the trove classifier issue when I try to `flit install` but don't see the Django dependency issue (in a fresh virtualenv).

Happy to merge this change anyway but might be worth figuring out if that issue is caused by something else?

**Comment by tm-kn** [comment]
> I see the trove classifier issue when I try to `flit install` but don't see the Django dependency issue (in a fresh virtualenv).
> 
> Happy to merge this change anyway but might be worth figuring out if that issue is caused by something else?

I think the Django issue must have been me using my global flit rather than the one in the virtual env. You are right, will roll back that change.

**Comment by tm-kn** [comment]
@tomusher I've rebased that commit now.

**Comment by tomusher** [comment]
Thanks @tm-kn

File: tox.ini [hunk 1]
```diff
@@ -44,6 +44,7 @@ basepython = python3.10
 
 deps =
     wagtail>=5.1
+    .[testing]
```
Comment (zerolab) [inline tox.ini:47]: note to self: this could be 
`extras = testing` ([reference](https://tox.wiki/en/4.11.3/config.html#extras))

but this works

Comment (tm-kn) [reply tox.ini:47]: Just followed what was in the `[testenv]` -> `deps`.

Comment (zerolab) [reply tox.ini:47]: yup. and that was good call to be consistent.

this is for me to follow up on ;)

**Pull request merged**